### PR TITLE
finish sign up (1)

### PR DESCRIPTION
**Pull request by sangpt**

### CODE REVIEW

**Comment by sangpt** [comment]
------------------------------------------
[o] Build Success!
------------------------------------------

[o] reek: success
[o] rubocop: success
[o] rails_best_practices: success
[o] bundle-audit: success
[o] brakeman: success
[o] scss-lint: success
[o] eslint: success
[!] rspec: failed but ignored

**Comment by sangpt** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -3,9 +3,26 @@
...
+            <li class="divider"></li>
+            <li>
+              <%= link_to t(".sign_out"), sign_out_path, method: :delete %>
+            </li>
+          </ul>
+        </li>
+        <% else %>
+        <li><%= link_to t(".users"), "#" %></li>
+        <li><%= link_to t(".help"), help_path %></li>
+        <li><%= link_to t(".log_in"), sign_in_path %></li>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header.html.erb]: indent from line 22-24

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,15 @@
-<% provide :title, t(".title") %>
-<h1><%= t ".title" %></h1>
+<div class="row">
+  <div class="col-md-6">
+    <div class="intro">
+      <%= t ".intro" %>.
+    </div>
+    <%= image_tag "social.png", alt: "social network", class: "img-responsive" %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/home.html.erb]: I18n

**Comment by huongnguyenmta** [comment]
commented

**Comment by sangpt** [comment]
ready

**Pull request merged**